src/components/learner-credit-management/AssignmentEnrollByDateCell.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import {
+  Icon, IconButtonWithTooltip, Stack,
+} from '@openedx/paragon';
+import { Warning } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { ENROLL_BY_DATE_DAYS_THRESHOLD, formatDate } from './data';
+import { isTodayWithinDateThreshold } from '../../utils';
+
+const ExpiringIconButtonWithToolTip = () => {
+  const intl = useIntl();
+  const internationalizedToolTipContent = intl.formatMessage({
+    id: 'lcm.budget.detail.page.assignments.table.columns.enroll.by.date.data.tool.tip.content',
+    defaultMessage: 'Enrollment deadline approaching',
+    description: 'On hover tool tip message for an upcoming enrollment date',
+  });
+  return (
+    <IconButtonWithTooltip
+      variant="warning"
+      tooltipContent={internationalizedToolTipContent}
+      tooltipPlacement="left"
+      src={Warning}
+      iconAs={Icon}
+      size="inline"
+      data-testid="upcoming-allocation-expiration-tooltip"
+    />
+  );
+};
+
+const AssignmentEnrollByDateCell = ({ row }) => {
+  const { original: { earliestPossibleExpiration: { date } } } = row;
+
+  const formattedEnrollByDate = formatDate(date);
+  const isDateWithinThreshold = isTodayWithinDateThreshold(
+    {
+      days: ENROLL_BY_DATE_DAYS_THRESHOLD,
+      date,
+    },
+  );
+
+  return (
+    <Stack direction="horizontal" gap={1}>
+      {isDateWithinThreshold && <ExpiringIconButtonWithToolTip />}
+      <div className="align-content-center">
```

Comment:
[curious] Does this `align-content-center` make a difference? By default, I don't believe this `div` is using flexbox, where the `align-content-center` is essentially a no-op here.

Is the vertical centering necessary here? `Stack` (in the "horizontal" direction) should be ensuring `align-items: center` on it's own I believe ([source](https://github.com/openedx/paragon/blob/4abe68b1cf60be6930d2771aa3c97b875ebed35b/src/Stack/index.scss#L27))?

Reply:
Removed. It may have been a remnant of centering the column cell and I forgot to remove it once the decision was made to keep the design in parity with the rest of the column patterns 👍🏽 .